### PR TITLE
ERM-1655: No results found message for simple search widget

### PR DESCRIPTION
**Pull request by MonireRasouli**

### CODE REVIEW

File: src/components/TypeComponents/SimpleSearch/Widget/SimpleSearch.js [hunk 1]
```diff
@@ -82,12 +82,16 @@ const SimpleSearch = ({
           <FormattedMessage id="ui-dashboard.simpleSearch.widget.nFoundBadge" values={{ total: data?.total }} />
         </Badge>
       </div>
-      <SimpleTable
-        key={`simple-table-${widget.id}`}
-        columns={columns}
-        data={useMemo(() => data?.results || [], [data])}
-        widgetId={widget.id}
-      />
+      {data?.results?.length === 0 ? (
```
Comment (EthanFreestone) [inline src/components/TypeComponents/SimpleSearch/Widget/SimpleSearch.js:85]: Here you can remove the `=== 0` and let truthiness deal with it :)

File: src/components/TypeComponents/SimpleSearch/Widget/SimpleSearch.js [hunk 2]
```diff
@@ -82,12 +82,16 @@ const SimpleSearch = ({
...
+        <FormattedMessage id="ui-dashboard.simpleSearch.widget.noResultFound" />
+      ) : (
```
Comment (EthanFreestone) [inline src/components/TypeComponents/SimpleSearch/Widget/SimpleSearch.js:87]: These shouldn't need brackets, can you remove them?

Should just be
```
condition ? <A /> : <B />
```

Comment (EthanFreestone) [reply src/components/TypeComponents/SimpleSearch/Widget/SimpleSearch.js:87]: Double check it works ofc

File: translations/ui-dashboard/en_US.json [hunk 3]
```diff
@@ -115,5 +115,7 @@
     "simpleSearch.widget.linkTextForWidget": "{linkText} for widget: {widgetName}",
     "dashboard.deleteWidget": "Delete widget",
     "dashboard.delete": "Delete",
-    "dashboard.deleteWidgetConfirmMessage": "Widget <strong>{name}</strong> will be <strong>deleted</strong>."
+    "dashboard.deleteWidgetConfirmMessage": "Widget <strong>{name}</strong> will be <strong>deleted</strong>.",
+    "simpleSearch.widget.noResultFound": "No results found"
```
Comment (EthanFreestone) [inline translations/ui-dashboard/en_US.json:119]: When adding a new translation we don't update the en_US file, only when changing an existing one. Can you remove this change please?

**Review by EthanFreestone — CHANGES_REQUESTED** [review]

File: translations/ui-dashboard/en_US.json (hunk 3 above)
Comment (MonireRasouli) [reply translations/ui-dashboard/en_US.json:119]: sure

**Review by EthanFreestone — APPROVED** [review]

**Pull request merged**